### PR TITLE
Palette self-hosted airgap refactor

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR refactors the Palette Self-Hosted AirGap mode install steps.

## Review Changes

💻 [Preview](https://deploy-preview-1230--docs-spectrocloud.netlify.app/enterprise-version/air-gap-repo)

🎫 [DOC-745](https://spectrocloud.atlassian.net/browse/DOC-745)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1230#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1230#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md:137]: Do app-name and service-name need tick marks or boldface?

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:57]: - What was phase 2?

Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:57]:
```suggestion
This guide focuses on the first installation phase, as the remaining ones are covered in the [Migrate Cluster to Enterprise](/deploying-an-enterprise-cluster) guide and the [Install Using Quick-Start Mode](/enterprise-version/deploying-the-platform-installer) guide.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:62]:
```suggestion
* The following minimum resources are required to deploy Palette.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:107]:
```suggestion
6. Review the details page. You may get a warning message stating the certificate is not trusted, you can click **Ignore** the message and select **Next**.
```

Comment (ritawatson) [reply content/docs/12-enterprise-version/06-air-gap-repo.md:107]: I modified this slightly because the user has to click the link to ignore it. Certificate errors could happen if they don't click Ignore, I think.

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:126]: Should parameter names be boldfaced (since they're in a form). I boldfaced parm names in the QS doc.

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:120]:
```suggestion
  | Encoded user-data | In order to fit into an xml attribute, this value is base64 encoded. This value will be decoded, and then processed normally as user-data. | - |
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:121]:
```suggestion
  | ssh public keys | This field is optional but indicates that the instance should populate the default user's `authorized_keys` with the provided public key. | -|
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:122]:
```suggestion
  | Default User's password|  Setting this value allows password-based login. The password will be good for only a single login.  If set to the string `RANDOM` then a random password will be generated, and written to the console. | - |
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:125]:
```suggestion
  |URL to seed instance data from| This field is optional but indicates that the instance should 'seed' user-data and meta-data from the given URL.| -|
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:217]:
```suggestion
  You must download the following three resources. Our support team will provide you with the credentials and download URL.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:361]:
```suggestion
17. If you will be using Edge deployments, go ahead and download the packages your Edge deployments will need. If you are not planning to use Edge, skip to end. You can come back to this step in the future and add the packages if needed. Click on the `...` tab for additional options.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:644]: Add a break in here to space before step 1.

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:657]:
```suggestion
5. Log in with the user `ubuntu` and the user password you specified during the installation. If you are using SSH, use the following command, and ensure you specify the path to your SSH private key and replace the IP address with your appliance's static IP.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:674]: Do you want to name this 'Example Output' to match previous labels for example output?

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:687]: 'Example Output'

File: content/docs/19-kubernetes-knowlege-hub.md
Comment (ritawatson) [inline content/docs/19-kubernetes-knowlege-hub.md:19]:
```suggestion
Welcome to the Spectro Cloud Kubernetes Knowledge Hub. You will find core Kubernetes tutorials, how-tos, frequently asked questions, and community-curated resources.
```

File: content/docs/19-kubernetes-knowlege-hub.md
Comment (ritawatson) [inline content/docs/19-kubernetes-knowlege-hub.md:21]:
```suggestion
If you have a topic in mind you would like to see, use the Feedback app at lower right.
```

**Review by ritawatson — APPROVED** [review]
Reviewed pretty much up to airgap steps. Made a few suggestions. I'll use some of the verbiage that applies in docs I'm working on so we're consistent. 

One suggestion for the great diagram. If the yellow numbered steps were on the same plane, it will make it visually easier to follow the steps - less bouncy for lack of a better term.  :)

**Review by ritawatson — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1230#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1230#pullrequestreview-1386228298" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:641]:
```suggestion
# Validation
```

**Review by ritawatson — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1230#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1230#pullrequestreview-1386233460" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:644]:
```suggestion

<br /
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:687]:
```suggestion
  Example Output:
```

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1230#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1230#pullrequestreview-1386233460" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:38]:
```suggestion
The airgap installation can be simplified into five major phases. 
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:57]:
```suggestion
This guide focuses on the first two installation phases, as the remaining ones are covered in the [Migrate Cluster to Enterprise](/deploying-an-enterprise-cluster) guide and the [Install Using Quick-Start Mode](/enterprise-version/deploying-the-platform-installer) guide.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:42]:
```suggestion
1. Download the Open Virtual Appliance (OVA) image and deploy the instance hosting the private repository that supports the airgap environment.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:35]:
```suggestion
![An architecture diagram outlining the five different install phases](/enterprise-version_air-gap-repo_overview-order-diagram.png)
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:125]:
```suggestion

  | Parameter | Description |  Default Value |
  | --- | --- | -- |
  | **Encoded user-data** | In order to fit into an XML attribute, this value is base64 encoded. This value will be decoded, and then processed normally as user-data. | - |
  | **ssh public keys** | This field is optional but indicates that the instance should populate the default user's `authorized_keys` with the provided public key. | -|
  | **Default User's password** |  Setting this value allows password-based login. The password will be good for only a single login.  If set to the string `RANDOM` then a random password will be generated, and written to the console. | - |
  | **A Unique Instance ID for this instance** |  Specifies the instance id. This is required and used to determine if the machine should take "first boot" actions| `id-ovf`|
  | **Hostname** | Specifies the hostname for the appliance. | `ubuntuguest` |
  | **URL to seed instance data from** | This field is optional but indicates that the instance should 'seed' user-data and meta-data from the given URL.| -|
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:211]:
```suggestion
16. The next set of steps will download the required binaries to support a Palette installation, such as the Palette Installer, required Kubernetes packages, and kubeadm pages. You can download these artifacts from the instance, or externally and transfer them to the instance. Click on each tab for further guidance.
```

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (zulfilee) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:211]: Kubernetes packages, and kubeadm packages.

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (zulfilee) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:233]: airgap-v3.3.15.bin . K8s prefix is only for K8s packages

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (zulfilee) [inline content/docs/12-enterprise-version/06-air-gap-repo.md:685]: Can we mask the password here and just state the password obtained from airgap-setup.sh 
We dont need to publicize that the password is a hard coded one and available for anyone to view

File: content/docs/12-enterprise-version/06-air-gap-repo.md
Comment (karl-cardenas-coding) [reply content/docs/12-enterprise-version/06-air-gap-repo.md:211]: Nice catch

**Pull request merged**
